packages/mgt-chat/src/statefulClient/StatefulGraphChatListClient.ts

```diff
@@ -587,26 +581,25 @@ class StatefulGraphChatListClient implements StatefulClient<GraphChatListClient>
    * @private
    * @memberof StatefulGraphChatListClient
    */
-  private async updateUserSubscription() {
-    // avoid subscribing to a resource with an empty userId
-    if (this.userId) {
-      // reset state to initial
-      this.notifyStateChange((draft: GraphChatListClient) => {
-        draft.status = 'initial';
-      });
-      // Subscribe to notifications for messages
-      this.notifyStateChange((draft: GraphChatListClient) => {
-        draft.status = 'creating server connections';
-      });
-      try {
-        await this._notificationClient.subscribeToUserNotifications(this._userId);
-      } catch (e) {
-        error('Failed to load chat data or subscribe to notications: ', e);
-        if (e instanceof GraphError) {
-          this.notifyStateChange((draft: GraphChatListClient) => {
-            draft.status = 'no messages';
-          });
-        }
+  private async updateUserSubscription(userId: string) {
+    if (userId === '') return;
+
+    // reset state to initial
```

Comment:
The call to clearCurrentUserMessages before the update already sets draft.status to initial. Does it need to happen again? Remove?

Reply:
Don't worry about this. When the client is newed up, clearCurrentUserMessage isn't called.